Bottom: 1f44e04c9253c33ec0ff494d90719c27555b5247
Top:    78e72e6bb3eee18ca8303381c6f797a4b8259127
Author: Andrew Cooper <andrew.cooper@novatechweb.com>
Date:   2018-08-16 09:17:44 -0500

Enable file-based secrets provider


---
diff --git a/roles/buildbot-master/files/master.cfg b/roles/buildbot-master/files/master.cfg
index 11942aaa..c453a09b 100644
--- a/roles/buildbot-master/files/master.cfg
+++ b/roles/buildbot-master/files/master.cfg
@@ -5,7 +5,7 @@
 # 'master.cfg' in your buildmaster's base directory.
 
 import os
-from buildbot.plugins import steps, util
+from buildbot.plugins import secrets, steps, util
 
 os.environ['LONG_RUN_TIMEOUT'] = '14400'
 
@@ -22,6 +22,9 @@ c['change_source'] = []
 # --master option)
 c['protocols'] = {"pb": {"port": int(os.getenv('BUILDBOT_WORKER_PORT', 9989))}}
 
+# SECRETS MANAGEMENT
+c['secretsProviders'] = [secrets.SecretInAFile(dirname="/run/secrets")]
+
 # STATUS TARGETS
 
 # 'status' is a list of Status Targets. The results of each build will be
